name = "PharException";
id = "pharexception"
alias[] = ""
description = " DivisionByZeroError is thrown when an attempt is made to divide a number by zero."
code = "<?php
try {
    echo 1/$a;
}
catch(DivisionByZeroError $e){
    echo "Trying to divide by 0";
}
?>"
documentation = "https://www.php.net/manual/en/class.pharexception.php"
analyzers[] = "Exceptions/CouldUseTry"
cobblers[] = ""
;phpVersionSince = "7.0"
;phpVersionUntil = "9.0"
related[] = "exception"
